### PR TITLE
Add get slashing endpoints

### PR DESCRIPTION
**Pull request by shayzluf**
[Part of] #3133

---

# Description

add endpoints for retrieval of slashing offences

### CODE REVIEW

File: proto/beacon/rpc/v1_gateway/services.pb.go [hunk 1]
```diff
@@ -0,0 +1,2216 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```
Comment (nisdas) [inline proto/beacon/rpc/v1_gateway/services.pb.go]: do we need this ? not sure if its needed

File: proto/slashing/slashing.proto [hunk 2]
```diff
@@ -88,3 +87,18 @@ message AttestationHistory {
     map<uint64, uint64> target_to_source = 1;
     uint64 latest_epoch_written = 2;
 }
+
+message SlashingStatusRequest {
+    enum SlashingStatus {
+        // Unknown default status in case it is not set
+        Unknown = 0;
+        // Active slashing proof hasn't been included yet.
+        Active=1;
```
Comment (rauljordan) [inline proto/slashing/slashing.proto]:
```suggestion
        Active = 1;
```

File: proto/slashing/slashing.proto [hunk 3]
```diff
@@ -88,3 +87,18 @@ message AttestationHistory {
...
+        // Included slashing proof that has been included in a block.
+        Included=2;
```
Comment (rauljordan) [inline proto/slashing/slashing.proto]:
```suggestion
        Included = 2;
```

File: proto/slashing/slashing.proto [hunk 4]
```diff
@@ -88,3 +87,18 @@ message AttestationHistory {
...
+        // Reverted slashing proof that has been reverted and therefore is relevant again.
+        Reverted=3; //relevant again
```
Comment (rauljordan) [inline proto/slashing/slashing.proto]:
```suggestion
        Reverted = 3; 
```

File: proto/beacon/rpc/v1_gateway/services.pb.go (hunk 1 above)
Comment (rauljordan) [reply proto/beacon/rpc/v1_gateway/services.pb.go]: it's nice to have at least for some native Go support for people that use it, we can leave it. The comments are added automatically by proto-gen-go

**Pull request merged**